guides/common/modules/ref_host-parameter-hierarchy.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Host_Parameter_Hierarchy_{context}"]
+= Host Parameter Hierarchy
+
+You can access host parameters when provisioning hosts.
+Hosts inherit their parameters from the following locations, in order of increasing precedence:
+
+[cols="50%,50%",options="header"]
+|====
+| Parameter Level | Set in {ProjectWebUI}
+| *Globally defined parameter* | *Configure* > *Global parameters*
+| *Organization-level parameters* | *Administer* > *Organizations*
+| *Location-level parameters* | *Administer* > *Locations*
+| *Domain-level parameters* | *Infrastructure* > *Domains*
+| *Subnet-level parameters* | *Infrastructure* > *Subnets*
+| *Operating system-level parameters* | *Hosts* > *Operating Systems*
+| *Host group-level parameters* | *Configure* > *Host Groups*
+| *Host parameters* | *Hosts* > *All Hosts*
```

Comment:
I suggest that specific parameter groups not be in bold because they are then not distinguished from the header. Same applies to the right column but since we always use bold for those locations I do not know how to resolve that. Italics maybe?

Reply:
Removed bold text in the first column.